src/Discord/WebSockets/Events/InteractionCreate.php

```diff
@@ -62,6 +62,22 @@ public function handle($data)
             $this->cacheUser($data->user);
         }
 
+        if (isset($data->entitlements)) {
+            foreach($data->entitlements as $entitlement) {
+                if ($entitlementPart = $this->discord->application->entitlements->get('id', $entitlement->id)) {
+                    $entitlementPart->fill((array) $entitlement);
+                } else {
+                    $this->discord->application->entitlements->set($entitlement->id, $this->discord->application->entitlements->create($entitlement, true));
```

Comment:
Since it's not available in my region, I just want to be sure that the entitlement received here is ONLY the entitlement belongs to the bot's application, and not to other's application

Reply:
as per https://discord.com/developers/docs/monetization/entitlements#entitlement-resource thats already the case